src/contracts/pools/Router.ts

```diff
@@ -115,40 +108,6 @@ export class Router extends SmartContractWithAddress {
     return trxReceipt
   }
 
-  /**
-   * Remove a token from oceanTokens list, pools without baseToken in this list have a opf Fee
-   * @param {String} address
-   * @param {String} tokenAddress address to remove
-   * @return {Promise<TransactionReceipt>}
-   */
-  public async removeApprovedToken<G extends boolean = false>(
```

Comment:
are we sure about this removal. Cause there's still `addApprovedToken()`? Not really clear based on code comments or contract method call where exactly the token is added/removed

Reply:
this is not related to pools. This needs to stay.

Reply:
don't really remove anything from Router.